onnx/defs/nn/defs.cc

```diff
@@ -1542,4 +1534,73 @@ ONNX_OPERATOR_SET_SCHEMA(
         })
         .SetDoc(TfIdfVectorizer_ver9_doc));
 
+static const char* StringNormalizer_ver10_doc = R"DOC(
+StringNormalization performs string operations for basic cleaning. 
+This operator has only one input (denoted by X) and only one output 
+(denoted by Y). This operator first examines the elements in the X, 
+and remove elements specified in "stopwords" attribute. 
+Note that an implementation should sequentially remove "stopwords[0]," then "stopwords[1]," 
+and so on. After removing stop words, the intermediate result can be further lowercased, 
+uppercased, or just returned depending the "casechangeaction" attribute.
+)DOC";
+
+ONNX_OPERATOR_SET_SCHEMA(
+    StringNormalizer,
+    10,
+    OpSchema()
+        .Input(0, "X", "Strings to normalize", "T")
+        .Output(0, "Y", "Normalized strings", "T")
+        .TypeConstraint(
+            "T",
+            {"tensor(string)"},
+            "Input/Output is a UTF-8 string tensor")
+        .Attr(
+            "casechangeaction",
+            "string enum that cases output to be lowercased/uppercases/unchanged. Valid values are \"LOWER\", \"UPPER\", \"NONE\"",
+            AttributeProto::STRING)
+        .Attr(
+            "is_case_sensitive",
```

Comment:
No sure if we want
```suggestion
            "iscasesensitive",
```
@linkerzhang, any preference?

Reply:
I suggest the other way, see my comments inline. :)